azure-devops/azext_devops/dev/team/extension.py

```diff
@@ -0,0 +1,105 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.log import get_logger
+
+from azext_devops.dev.common.services import (get_extension_client,
+                                              resolve_instance)
+from azext_devops.dev.common.arguments import resolve_true_false
+
+logger = get_logger(__name__)
+
+
+def list_extensions(include_built_in=None, include_disabled=None, organization=None, detect=None):
```

Comment:
Why not add the default value here include_built_in='False'? Even documentation gets generated automatically then like Default Value: False

Reply:
done